content/en/docs/concepts/services-networking/service.md

```diff
@@ -1151,11 +1154,16 @@ spec:
     - name: http
       protocol: TCP
       port: 80
-      targetPort: 9376
+      targetPort: 49152
   externalIPs:
-    - 80.11.12.10
+    - 198.51.100.32
 ```
 
+{{< note >}}
+Kubernetes does not manage allocation of `externalIPs`; these are the responsibility
```

Comment:
Do you mean the actual reservation of those IPs? Or the allocation of those IPs in the manifest?

Reply:
Ah, that belongs in a different PR I think.